support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/response/accesstoken/ext/AccessTokenAuthorizationCodeGrantRequestExtractor.java

```diff
@@ -87,15 +86,6 @@ protected String getRegisteredServiceIdentifierFromRequest(final HttpServletRequ
         return request.getParameter(OAuth20Constants.REDIRECT_URI);
     }
 
-    /**
-     * Is allowed to generate refresh token ?
```

Comment:
What you have here is fine but I'd suggest to restore this bit and allow extensions to override this behavior regardless of the service config. This will make it easier to upgrade too for anyone who might have overriden this method.

Reply:
Sure, makes sense. Will take care of it!

Reply:
I restored this method. 
There seems to be an issue on travis-ci when it tries to install JDK for the javadoc build. Would it be possible to manually trigger a rebuild?

Reply:
Sure. I'll re-run the build.